modules/aws/asg_test.go

```diff
@@ -0,0 +1,160 @@
+package aws
+
+import (
+	"fmt"
+	"testing"
+	"time"
+
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/service/autoscaling"
+	"github.com/aws/aws-sdk-go/service/ec2"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+
+	"github.com/gruntwork-io/terratest/modules/random"
+)
+
+func TestGetCapacityInfoForAsg(t *testing.T) {
```

Comment:
Do we have an ASG example in the `?examples` folder? If so, perhaps add these tests to the corresponding tests in the `/tests` folder so users can discover this functionality more easily?